dataland-data-exporter/src/test/kotlin/org/dataland/datalanddataexporter/services/CsvExporterTest.kt

```diff
@@ -113,7 +112,33 @@ class CsvExporterTest {
     }
 
     @Test
-    fun `check that running the sfdr export does not throw an error`() {
+    fun `check that the sfdr export runs as expected`() {
         assertDoesNotThrow { csvDataExporter.exportSfdrData(outputDirectory = "./src/test/resources/csv/output") }
+        verify(mockMetadataControllerApi, times(1))
+            .getListOfDataMetaInfo(
+                dataType = DataTypeEnum.sfdr,
+            )
+        verify(mockSfdrDataControllerApi, times(1))
+            .getCompanyAssociatedSfdrData(
+                dataId = any(),
+            )
+        verify(mockCompanyDataControllerApi, times(1))
+            .getCompanyById(
+                companyId = any(),
+            )
+    }
+
+    @Test
+    fun `check that running the sfdr export produces two csv files`() {
+        val outputDirectory = "./src/test/resources/csv/output"
+        val directory = File(outputDirectory)
+        if (directory.exists()) {
+            directory.deleteRecursively()
+        }
+
+        csvDataExporter.exportSfdrData(outputDirectory = outputDirectory)
+
+        val filesInDirectory = File(outputDirectory).listFiles()
+        assertTrue((filesInDirectory?.size ?: 0) >= 2, "There should be at least two new csv-files.")
```

Comment:
```suggestion
        assertNotNull(filesInDirectory, "There should be csv-files.")
        assertTrue((filesInDirectory!!.size) == 2, "There should be exactly two new csv-files.")
```

if you check for '>=', adapt test description accordingly